### ISSUE TITLE
[Bug]: Distant Horizons Bug

### ISSUE THREAD

**Issue by DavWils** [nonamecrackers2/crackers-wither-storm-mod#2235]
### Guidelines

- [X] By checking this box, I am saying that I have read the guidelines and believe to be following them to the best of my ability

### Minecraft Version

1.20.1

### Mod Version

4.2.1

### Mod Loader

Forge

### Mod Loader Version

47.3.0

### Describe the bug

When Distant Horizons is installed, Witherstorm won't follow player.

### Steps to Reproduce

Install Witherstorm mod & Distant Horizons, then summon Witherstorm, run far away (to the point where the Witherstorm is far enough to where the chunk would normally be unloaded). Leave, rejoin, and then the Witherstorm won't follow you until you load the chunk it's in.

### Crash Reports/Logs

No Crash

### Is OptiFine installed?

- [ ] OptiFine is installed

### Mod List

[Craftable Horse Armour & Saddle [CHA&S] (by EwyBoy)](https://www.curseforge.com/minecraft/mc-mods/cha-s)
[Horse Combat Controls (by pitbox46)](https://www.curseforge.com/minecraft/mc-mods/horse-combat-controls)
[Tectonic (by Apollo)](https://www.curseforge.com/minecraft/mc-mods/tectonic)
[Immersive Armors [Fabric/Forge] (by Conczin)](https://www.curseforge.com/minecraft/mc-mods/immersive-armors)
[Explorer's Compass (by Chaosyr)](https://www.curseforge.com/minecraft/mc-mods/explorers-compass)
[Bliss Shaders (by Xonk___)](https://www.curseforge.com/minecraft/shaders/bliss-shader)
[Better Combat [Fabric & Forge] (by daedelus_dev)](https://www.curseforge.com/minecraft/mc-mods/better-combat-by-daedelus)
[Deeper and Darker (by KyaniteMods)](https://www.curseforge.com/minecraft/mc-mods/deeperdarker)
[Simply Swords [Fabric & Forge] (by sweenuss)](https://www.curseforge.com/minecraft/mc-mods/simply-swords)
[Curios API (Forge/NeoForge) (by TheIllusiveC4)](https://www.curseforge.com/minecraft/mc-mods/curios)
[Clumps (by Jaredlll08)](https://www.curseforge.com/minecraft/mc-mods/clumps)
[Callable Horses (by Tschipp)](https://www.curseforge.com/minecraft/mc-mods/callable-horses)
[Shiny Horses Forge - Enchantable Horse Armor (by tinytransfem)](https://www.curseforge.com/minecraft/mc-mods/shiny-horses-forge-enchantable-horse-armor)
[Aquaculture 2 (by Shadow)](https://www.curseforge.com/minecraft/mc-mods/aquaculture)
[Puzzles Lib [Forge & Fabric] (by Fuzs)](https://www.curseforge.com/minecraft/mc-mods/puzzles-lib)
[[EMF] Entity Model Features [Fabric & Forge] (by traben_0)](https://www.curseforge.com/minecraft/mc-mods/entity-model-features)
[Cloth Config API (Fabric/Forge/NeoForge) (by shedaniel)](https://www.curseforge.com/minecraft/mc-mods/cloth-config)
[Comforts (Fabric/Forge/Quilt) (by TheIllusiveC4)](https://www.curseforge.com/minecraft/mc-mods/comforts)
[Caelus API (Forge/NeoForge) (by TheIllusiveC4)](https://www.curseforge.com/minecraft/mc-mods/caelus)
[Visual Workbench [Forge & Fabric] (by Fuzs)](https://www.curseforge.com/minecraft/mc-mods/visual-workbench)
[Easy Anvils [Forge & Fabric] (by Fuzs)](https://www.curseforge.com/minecraft/mc-mods/easy-anvils)
[playerAnimator (by KosmX)](https://www.curseforge.com/minecraft/mc-mods/playeranimator)
[Not Enough Animations (by tr7zw)](https://www.curseforge.com/minecraft/mc-mods/not-enough-animations)
[Fresh Animations (by FreshLX)](https://www.curseforge.com/minecraft/texture-packs/fresh-animations)
[Nullscape (by Starmute)](https://www.curseforge.com/minecraft/mc-mods/nullscape)
[Mouse Tweaks (by YaLTeR)](https://www.curseforge.com/minecraft/mc-mods/mouse-tweaks)
[Terralith (by Starmute)](https://www.curseforge.com/minecraft/mc-mods/terralith)
[Jade 🔍 (by Snownee)](https://www.curseforge.com/minecraft/mc-mods/jade)
[Patchouli (by Vazkii)](https://www.curseforge.com/minecraft/mc-mods/patchouli)
[Paragliders (by Tictim)](https://www.curseforge.com/minecraft/mc-mods/paragliders)
[Quark (by Vazkii)](https://www.curseforge.com/minecraft/mc-mods/quark)
[Architectury API (Fabric/Forge/NeoForge) (by shedaniel)](https://www.curseforge.com/minecraft/mc-mods/architectury-api)
[Forgified Fabric API (by Su5eD)](https://www.curseforge.com/minecraft/mc-mods/forgified-fabric-api)
[Bookshelf (by DarkhaxDev)](https://www.curseforge.com/minecraft/mc-mods/bookshelf)
[Prism [Neo/Forge] (by Grend_G)](https://www.curseforge.com/minecraft/mc-mods/prism-lib)
[Elytra Slot (Fabric/Forge/Quilt) (by TheIllusiveC4)](https://www.curseforge.com/minecraft/mc-mods/elytra-slot)
[Legendary Tooltips [Neo/Forge] (by Grend_G)](https://www.curseforge.com/minecraft/mc-mods/legendary-tooltips)
[Farmer's Delight (by vectorwing)](https://www.curseforge.com/minecraft/mc-mods/farmers-delight)
[Nature's Compass (by Chaosyr)](https://www.curseforge.com/minecraft/mc-mods/natures-compass)
[Cracker's Wither Storm Mod (by nonamecrackers2)](https://www.curseforge.com/minecraft/mc-mods/crackers-wither-storm-mod)
[Enchantment Descriptions (by DarkhaxDev)](https://www.curseforge.com/minecraft/mc-mods/enchantment-descriptions)
[Oculus (by Asek3)](https://www.curseforge.com/minecraft/mc-mods/oculus)
[AppleSkin (by squeek502)](https://www.curseforge.com/minecraft/mc-mods/appleskin)
[Easy Magic [Forge & Fabric] (by Fuzs)](https://www.curseforge.com/minecraft/mc-mods/easy-magic)
[[ETF] Entity Texture Features - [Fabric & Forge] (by traben_0)](https://www.curseforge.com/minecraft/mc-mods/entity-texture-features-fabric)
[Iceberg [Neo/Forge] (by Grend_G)](https://www.curseforge.com/minecraft/mc-mods/iceberg)
[Zeta (by Vazkii)](https://www.curseforge.com/minecraft/mc-mods/zeta)
[Distant Horizons: A Level of Detail mod (by James_Seibel)](https://www.curseforge.com/minecraft/mc-mods/distant-horizons)
[Just Enough Items (JEI) (by mezz)](https://www.curseforge.com/minecraft/mc-mods/jei)
[Enhanced Boss Bars (by Nakuring)](https://www.curseforge.com/minecraft/mc-mods/enhanced-boss-bars)
[Embeddium (by FiniteReality)](https://www.curseforge.com/minecraft/mc-mods/embeddium)
[Potato Shader (by RRe36)](https://www.curseforge.com/minecraft/shaders/potato-shaders)

**Comment by TSEG6** [comment]
Already known

**Comment by TSEG6** [comment]
This has nothing to do with distant horizons, the wither storm seems like it's distracted based on your description. Since it'll become focused when approached. And that's exactly what you said happened.

**Comment by nonamecrackers2** [comment]
Yeah, confirm if this is the case by spawning a fresh Wither Storm and seeing if this bug still occurs.
